open_bus_gtfs_etl/cli.py

```diff
@@ -48,3 +48,7 @@ def load_to_db(**kwargs):
 def cleanup_dated_paths(**kwargs):
     """Delete old directories from the dated paths"""
     api.cleanup_dated_paths(**kwargs)
+
+
+if __name__ == '__main__':
```

Comment:
why did you add this? I removed it in this PR: https://github.com/hasadna/open-bus-gtfs-etl/pull/7/files#diff-02fcca231fb487d43a4a0e2f5eda40ce5be7e3087c678a13db9fbbdcfad30148L92

we should have a single entrypoint from the click CLI, otherwise it may cause code discrepencies